alcs-frontend/src/app/features/board/board.component.ts

```diff
@@ -507,7 +507,19 @@ export class BoardComponent implements OnInit, OnDestroy {
 
   private mapApplicationDecisionConditionToCard(
     applicationDecisionCondition: ApplicationDecisionConditionCardBoardDto,
-  ): CardData {
+  ): ConditionCardData {
+    let isExpired = false;
+    let isPastDue = false;
+
+    for (const condition of applicationDecisionCondition.conditions) {
+      if (condition.status === 'EXPIRED') {
```

Comment:
Super nitpicking, but for these situations, I always think it's a little cleaner to just do `isExpired = condition.status === 'EXPIRED'`. I think it's also more robust to use condition status enum directly: `isExpired = condition.status === CONDITION_STATUS.EXPIRED`